docs/deployment/contract-deployment-options.md

```diff
@@ -38,7 +37,7 @@ lspFactory.LSP3UniversalProfile.deploy({...}, {
 })
 ```
 
-Or if you want your custom bytecode to be deployed at the KeyManager contract itself rather than at the base contract implementation you can set `deployProxy` to false:
+Or if you want to use your custom bytecode for the KeyManager contract itself (rather than for a base contract behind a proxy), you can set deployProxy to `false`:
```

Comment:
For consistency and clarity, let's use either:
- "**Key Manager**" if referring to the Key Manager in general.
- `LSP6KeyManager` if referring to the **actual contract**